### PR TITLE
Improve desktop GUI folder nesting ui

### LINKED ISSUE #3652: Indentation of tests in the cypress electron app is poor
### Current behavior:

Indentation in the electron runner for test cases is poorly organised.

### Desired behavior:

To have a better layout for readability. Simply being nested correctly would help.

### Steps to reproduce: (app code and test code)

Simply create a deeply nested hierarchy of tests.

image

### Versions

3.1.5

### PR DESCRIPTION
**Pull request by desfero**
Fixes #3652

### CODE REVIEW

File: packages/desktop-gui/src/specs/specs.scss [hunk 1]
```diff
@@ -126,21 +131,45 @@
...
+      margin-right: 5px;
     }
 
     i.folder-collapse-icon {
       display: inline-block;
       margin-right: 4px;
     }
 
-    &>div>div {
-      padding-bottom: 0;
```
Comment (desfero) [inline packages/desktop-gui/src/specs/specs.scss]: hm, there is no `padding-bottom` anywhere so no need to reset? 🤔

**Comment by jennifer-shehane** [comment]
If you want to fix the last folders icon when the filename is really long, that'd be cool, but I wouldn't consider it part of the scope of this fix. 👍 	

Screen Shot 2019-03-18 at 1 15 20 PM

**Comment by desfero** [comment]
@jennifer-shehane thanks for the review, I will add some tests and fix long spec/folder name. 
You definitely should consider visual regression tests on top of a storybook for e.g. to cover this type of bugs IMO.

**Comment by desfero** [comment]
@jennifer-shehane done

**Comment by jennifer-shehane** [comment]
Thanks! ⭐️⭐️⭐️⭐️⭐️

I agree, some visual regression tests would be nice. 😥

**Pull request merged**